README.md

```diff
@@ -36,6 +36,9 @@ dependencies {
 
     // Detects the current platform and adds the corresponding Chromium binaries.
     implementation(jxbrowser.currentPlatform())
```

Comment:
Do we need to call both: currentPlatform and crossPlatform?

Reply:
No. In this example, we demonstrate all possible calls.